pysus/online_data/__init__.py

```diff
@@ -100,15 +99,29 @@ def _parse_dftypes(df: pd.DataFrame) -> pd.DataFrame:
     and converting dtypes into correct types.
     """
 
+    def map_column_func(column_names: list[str], func):
```

Comment:
This function can be replaced by the `applymap` method of pandas dataframes: 
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.applymap.html#pandas.DataFrame.applymap

Reply:
I've tried changing to applymap, but I couldn't do in a way that I can add more columns without needing to add more if statements for each column

Reply:
It seems that you are applying func  only to the columns of df  that are in `column_names`, so this function could be simplified to:
```python
df[[c for c in df.columns if c in column_names]].applymap(func)
```

Reply:
oh thats nice, thanks!